srcCpp/Infrastructure_pro.cxx

```diff
@@ -523,7 +523,11 @@ bool configureShmemTransport(
     DDS_Property_t *parentProp =
             DDSPropertyQosPolicyHelper::lookup_property(qos.property,
                     "dds.transport.shmem.builtin.parent.message_size_max");
-    int parentMsgSizeMax = atoi(parentProp->value);
+
+    int parentMsgSizeMax = DEFAULT_MESSAGE_SIZE_MAX;
```

Comment:
@fercsrti77, we agreed we needed to review this part of the code and decide what numbers to set for the buffers.